.gitignore

```diff
@@ -11,10 +11,10 @@ doc/
 lib/bundler/man
 pkg
 rdoc
-test/reports/*
+test/reports
 test/tmp
 test/version_tmp
 tmp
-test/test.sqlite3
```

Comment:
do we no longer have this file locally to ignore?

Reply:
We never needed this one specifically because all `.sqlite3` files are ignored on line 3 with `*.sqlite3`